### PR TITLE
Add warning for unsupported compose yaml fields

### PR DESCRIPTION
**Pull request by uttarasridhar**
$ make build

$ ./bin/local/ecs-cli compose up
WARN[0000] Skipping unsupported YAML option...           option name=volumes
WARN[0000] Skipping unsupported YAML option...           option name=networks
WARN[0000] Skipping unsupported YAML option for service...  option name=build service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=cap_add service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=cap_drop service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=cpuset service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=cpu_quota service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=cgroup_parent service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=container_name service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=devices service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=depends_on service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=expose service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=external_links service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=ipc service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=mac_address service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=memswap_limit service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=network_mode service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=networks service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=pid service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=shm_size service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=stop_signal service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=restart service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=stdin_open service name=web
WARN[0000] Skipping unsupported YAML option for service...  option name=tty service name=web

### CODE REVIEW

File: ecs-cli/modules/compose/ecs/utils/convert_task_definition.go [hunk 1]
```diff
@@ -36,6 +37,24 @@ const (
 	readWriteVolumeAccessMode = "rw"
 )
 
+// supported fields/options from compose YAML file
+var supportedComposeYamlOptions = [...]string{
```
Comment (aaithal) [inline ecs-cli/modules/compose/ecs/utils/convert_task_definition.go]: doesn't `[]string{` work here?

File: ecs-cli/modules/compose/ecs/utils/convert_task_definition.go [hunk 2]
```diff
@@ -68,6 +90,68 @@ func ConvertToTaskDefinition(taskDefinitionName string, context *project.Context
...
+	}
+}
+
+// logUnsupportedServiceConfigFields
+func logUnsupportedServiceConfigFields(serviceName string, config *config.ServiceConfig) {
+	configValue := reflect.ValueOf(config).Elem()
+	configType := configValue.Type()
+
+	for i := 0; i < configValue.NumField(); i++ {
+		f := configValue.Field(i)
```
Comment (aaithal) [inline ecs-cli/modules/compose/ecs/utils/convert_task_definition.go]: can you rename this to `field`?

File: ecs-cli/modules/compose/ecs/utils/convert_task_definition.go [hunk 3]
```diff
@@ -68,6 +90,68 @@ func ConvertToTaskDefinition(taskDefinitionName string, context *project.Context
...
+		ft := configType.Field(i)
```
Comment (aaithal) [inline ecs-cli/modules/compose/ecs/utils/convert_task_definition.go]: can you rename this to `fieldType`?

Comment (uttarasridhar) [inline ecs-cli/modules/compose/ecs/utils/convert_task_definition.go]: renamed to fieldType

File: ecs-cli/modules/compose/ecs/utils/convert_task_definition.go (hunk 2 above)
Comment (uttarasridhar) [inline ecs-cli/modules/compose/ecs/utils/convert_task_definition.go]: done

File: ecs-cli/modules/compose/ecs/utils/convert_task_definition.go (hunk 1 above)
Comment (uttarasridhar) [inline ecs-cli/modules/compose/ecs/utils/convert_task_definition.go]: it does. changed.

**Comment by aaithal** [comment]
:shipit:

**Comment by samuelkarp** [comment]
:+1:

**Pull request merged**